projects/plugins/crm/src/automation/class-attribute-definition.php

```diff
@@ -0,0 +1,261 @@
+<?php
+/**
+ * Attribute Definition
+ *
+ * @package automattic/jetpack-crm
+ * @since $$next-version$$
+ */
+
+namespace Automattic\Jetpack\CRM\Automation;
+
+/**
+ * Step Attribute.
+ *
+ * The Step Attribute represents
+ *
+ * @since $$next-version$$
+ */
+class Attribute_Definition {
+
+	/**
+	* Represents a dropdown selection input.
+	*
+	* @since $$next-version$$
+	* @var string
+	*/
+	const SELECT = 'select';
+
+	/**
+	* Represents a checkbox input.
+	*
+	* @since $$next-version$$
+	* @var string
+	*/
+	const CHECKBOX = 'checkbox';
+
+	/**
+	* Represents a textarea input.
+	*
+	* @since $$next-version$$
+	* @var string
+	*/
+	const TEXTAREA = 'textarea';
+
+	/**
+	* Represents a text input.
+	*
+	* @since $$next-version$$
+	* @var string
+	*/
+	const TEXT = 'text';
+
+	/**
+	* Represents a text input.
+	*
+	* @since $$next-version$$
+	* @var string
+	*/
+	const HIDDEN = 'hidden';
```

Comment:
For our use case, step attributes, the hidden should not be an input for the end user.

Reply:
Removed in 99ecac9.